dashboard/app/models/levels/level.rb

```diff
@@ -70,6 +70,7 @@ class Level < ApplicationRecord
     reference_links
     name_suffix
     parent_level_id
+    contained_level_names
```

Comment:
For my understanding -- this will stick around as the way to serialize the contained levels relationship but shouldn't be referenced outside of seeding right?

Reply:
Correct!